### PR TITLE
Fixes #13757 - spilt layout helper into two helpers

### PR DESCRIPTION
**Pull request by amirfefer**
Layout helper is too large, all the forms logic moved to form helper

### CODE REVIEW

**Comment by dLobatog** [comment]
@amirfefer Thanks, notice tests are failing

**Comment by amirfefer** [comment]
[test]

**Comment by tbrisker** [comment]
@amirfefer there are a few methods in `ApplicationHelper` which should also be moved to `FormHelper`

**Comment by amirfefer** [comment]
@tbrisker `link_to_remove_fields` and `link_to_add_fields` methods moved to `FormHelper`

File: app/helpers/form_helper.rb [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            end
+
+        input_group collection_select, button_part
+      else
+        collection_select
+      end
+    end
+  end
+
+  def addClass(options = {}, new_class = '')
```
Comment (tbrisker) [inline app/helpers/form_helper.rb:155]: This should stay in layout helper

**Comment by tbrisker** [comment]
@amirfefer Looks good, just one function you accidentally moved to `FormHelper`, will merge once you move it back.

File: app/helpers/form_helper.rb (hunk 1 above)
Comment (tbrisker) [inline app/helpers/form_helper.rb:155]: Ok, nevermind, looks like this is only used in the form helper.

**Comment by tbrisker** [comment]
Merged as 3952cb53d9893751db68a6cb320fa36fe47cf885, thanks @amirfefer!

**Pull request closed without merging**